backend/spotihue/spotihue.py

```diff
@@ -79,10 +99,19 @@ def _initialize_hue(self, hue_bridge_ip_address) -> phue.Bridge:
         Returns:
             phue.Bridge: Initialized Hue Bridge object.
         """
-        hue = phue.Bridge(hue_bridge_ip_address)
+        hue = phue.Bridge(hue_bridge_ip_address, config_file_path='.python_hue')
         hue.connect()
         return hue
 
+    def _get_current_track(self) -> Optional[dict]:
```

Comment:
Doc string? Want to have these in here because eventually I'll add auto-documentation.